terraform/digitalgov.gov.tf

```diff
@@ -404,7 +404,9 @@ resource "aws_route53_record" "digitalgov_gov_dmarc_digitalgov_gov_txt" {
   name    = "digitalgov.gov."
   type    = "TXT"
   ttl     = 300
-  records = ["${local.spf_no_mail}"]
+  records = [
+    "v=spf1 include:1962994m.challenge.gov ~all"
```

Comment:
Not a blocker, but we may want to use a [local](https://www.terraform.io/docs/configuration/locals.html) to ensure it's the same across all our HubSpot domains.

Reply:
Happy to add that in.